.github/workflows/ci.yml

```diff
@@ -124,10 +124,40 @@ jobs:
     steps:
       - run: exit 0
 
+  # Maybe launch it only in testnet branches
+  gas-costs-check:
+    if: github.ref != 'refs/heads/staging'
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+        with:
+          submodules: "recursive"
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly-2022-12-24
```

Comment:
nightly-latest ?

Reply:
Right now we fix the number on each job for each testnet to avoid compilation error on newest nightly we also have a `toolchain.toml` in the repository